railway.json

```diff
@@ -4,6 +4,6 @@
     "builder": "NIXPACKS"
   },
   "deploy": {
-    "startCommand": "/opt/venv/bin/app database upgrade --no-prompt && /opt/venv/bin/app run --wc 2 --host 0.0.0.0 --port $PORT"
+    "startCommand": "/opt/venv/bin/app database upgrade --no-prompt && /opt/venv/bin/app --app app.asgi:app run --wc 2 --host 0.0.0.0 --port $PORT"
```

Comment:
I'm curious why we need this.

This [line](https://github.com/litestar-org/litestar-fullstack/blob/main/src/app/__main__.py#L15) should be covering this for us.

Reply:
Maybe we should set the `LITESTAR_APP` var in the railway config as well.

Reply:
so you are right. its not needed. I just had my env set wrong. for the `LITESTAR_APP` i wasn't aware you could define variables in the railway.json config directly